flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogListStateTest.java

```diff
@@ -38,50 +38,48 @@
 
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** ChangelogListState Test. */
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ChangelogListStateTest {
```

Comment:
```suggestion
class ChangelogListStateTest {
```
could be package private